librz/bin/format/pe/pe.h

```diff
@@ -27,10 +60,26 @@ struct rz_bin_pe_section_t {
 	ut64 vsize;
 	ut64 vaddr;
 	ut64 paddr;
+	int flags;
```

Comment:
I see flags was changed back to an `int` from a `ut64` in [7e13f8eafc](https://github.com/PinkNoize/rizin/commit/7e13f8eafc4d83b498f2589e4eb91f8215bd467b)
C only guarantees that an int is at least 16 bits

Was this intentional and if so what is the reason for it?